boards/adi/max78002evkit/doc/index.rst

```diff
@@ -165,99 +120,324 @@ The ``max78002evkit/max78002/m4`` board target supports the following interfaces
-| JP35      | I2C1 SDA          | I2C1 DATA pull-up                                                                |
-+-----------+-------------------+----------------------------------------------------------------------------------+
-| JP36      | I2C1 SCL          | I2C1 CLOCK pull-up                                                               |
-+-----------+-------------------+----------------------------------------------------------------------------------+
-| JP37      | I2S CK SEL        | Select SMA connector J6 / onboard crystal oscillator for I2S master clock source |
-+-----------+-------------------+----------------------------------------------------------------------------------+
-| JP38      | DVP CAM PWR       | Enable/Disable OVM7692 for DVP camera PWDN input                                 |
-+-----------+-------------------+----------------------------------------------------------------------------------+
-| JP39      | SW CAM PWUP       | Camera reset and power up under port pin control                                 |
-+-----------+-------------------+----------------------------------------------------------------------------------+
-| JP40      | HW PWUP / SW PWUP | Camera will reset and power up as soon as 3.3V reaches a valid level             |
-+-----------+-------------------+----------------------------------------------------------------------------------+
-| JP41      | CSI CAM I2C EN    | Connect/Disconnect I2C1 to CSI camera Digilent P5C I2C                           |
-+-----------+-------------------+----------------------------------------------------------------------------------+
-| JP42      | TFT DC            | TFT data/command select connects to port 2.2                                     |
-+-----------+-------------------+----------------------------------------------------------------------------------+
-| JP43      | TFT CS            | Select port 0.3 / port 1.7 to drive TFT CS                                       |
-+-----------+-------------------+----------------------------------------------------------------------------------+
-| JP44      | LED1 EN           | Enable/Disable LED1                                                              |
-+-----------+-------------------+----------------------------------------------------------------------------------+
-| JP45      | LED2 EN           | Enable/Disable LED2                                                              |
-+-----------+-------------------+----------------------------------------------------------------------------------+
++-----------+-------------------+---------------+--------------------------------------------------------------------------------------------------+
+| Name      | Name              | Settings      | Description                                                                                      |
```

Comment:
Why does this table need to be in the Zephyr docs? Aren't you already referencing the board's datasheet?

Reply:
Yeah it just feels "too much", and unnecessarily clutters the Zephyr docs with something that is really just paraphrasing the board manual IMO